src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -655,6 +655,57 @@ void GetElectricEIRChillerInput(EnergyPlusData &state)
             }
         }
 
+        if (NumAlphas > 16) {
+            thisChiller.CondenserFlowControl = static_cast<DataPlant::CondenserFlowControl>(
+                getEnumValue(DataPlant::CondenserFlowControlNamesUC, state.dataIPShortCut->cAlphaArgs(17)));
+        } else {
+            thisChiller.CondenserFlowControl = DataPlant::CondenserFlowControl::ConstantFlow;
+        }
+
+        if (thisChiller.CondenserFlowControl == DataPlant::CondenserFlowControl::Invalid) {
+            ShowSevereError(state,
+                            format("{}{}=\"{}\",", RoutineName, state.dataIPShortCut->cCurrentModuleObject, state.dataIPShortCut->cAlphaArgs(1)));
+            ShowContinueError(state, format("Invalid {}={}", state.dataIPShortCut->cAlphaFieldNames(17), state.dataIPShortCut->cAlphaArgs(17)));
+            ShowContinueError(state, "Available choices are ConstantFlow, ModulatedChillerPLR, ModulatedLoopPLR, or ModulatedDeltaTemperature");
+            ShowContinueError(state, "Flow mode ConstantFlow is assumed and the simulation continues.");
+            thisChiller.CondenserFlowControl = DataPlant::CondenserFlowControl::ConstantFlow;
+        };
```

Comment:
I'm not sure....could it get to this IF block?  Will the input processor protect the inputs?  It always feels like there are wrinkles in what IP will populate, so I'm not sure, and this is fine.  Maybe @mjwitte disagrees...